tests/helper.py

```diff
@@ -26,9 +28,13 @@ def relative_image_difference(img1: torch.Tensor, img2: torch.Tensor) -> torch.T
 
 
 def dotproduct_adjointness_test(
-    operator: Operator, u: torch.Tensor, v: torch.Tensor, relative_tolerance: float = 1e-3, absolute_tolerance=1e-5
+    linear_operator: LinearOperator,
```

Comment:
not to fond of this change.
that it is a linear_operator is already given by the type info.
does it ake the code easier to understand to read linear_operator than operator?

but if you have strong preference for linear_operator here, I wont care.